org.eclipse.xtext.xbase.testlanguages/src-gen/org/eclipse/xtext/xbase/testlanguages/bug462047/services/Bug462047LangGrammarAccess.java

```diff
@@ -475,8 +475,8 @@ public ParserRule getXListLiteralRule() {
 	//XClosure XExpression:
 	//	=> ({XClosure}
 	//	'[')
-	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)?
-	//	explicitSyntax?='|')?
+	//	=> ((declaredFormalParameters+=JvmFormalParameter (',' declaredFormalParameters+=JvmFormalParameter)*)
+	//	? explicitSyntax?='|')?
```

Comment:
Still weird @ArneDeutsch

Reply:
The formatter does not do that. Unsure why it happens here.